### PR TITLE
Empty M3 template

### PR DESCRIPTION
**Pull request by android-dev-lxl**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,30 +1,15 @@
-/*
```
Comment (kkuan2011) [inline app/build.gradle:1]: can we leave the license header in? and for subsequent files too

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,31 +1,21 @@
...
-  -->
-
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
 
     <application
         android:allowBackup="true"
+        android:dataExtractionRules="@xml/data_extraction_rules"
```
Comment (kkuan2011) [inline app/src/main/AndroidManifest.xml:7]: you could remove these two lines since we won't need them

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,31 +1,21 @@
...
+        android:fullBackupContent="@xml/backup_rules"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.Unscramble">
+        android:theme="@style/Theme.Unscramble"
+        tools:targetApi="31">
```
Comment (kkuan2011) [inline app/src/main/AndroidManifest.xml:14]: update?

File: app/src/main/java/com/example/unscramble/ui/theme/Theme.kt [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    secondary = PurpleGrey80,
+    tertiary = Pink80
+)
+
+private val LightColorScheme = lightColorScheme(
+    primary = Purple40,
+    secondary = PurpleGrey40,
+    tertiary = Pink40
+
+    /* Other default colors to override
```
Comment (kkuan2011) [inline app/src/main/java/com/example/unscramble/ui/theme/Theme.kt:29]: you can remove any commented out code

File: app/src/main/res/xml/data_extraction_rules.xml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (kkuan2011) [inline app/src/main/res/xml/data_extraction_rules.xml:1]: can remove this file

**Review by kkuan2011 — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 6]
```diff
@@ -1,32 +1,6 @@
...
 plugins {
-    id 'com.android.application' version '7.4.0' apply false
-    id 'com.android.library' version '7.4.0' apply false
-    id 'org.jetbrains.kotlin.android' version '1.8.0' apply false
-}
-
-task clean(type: Delete) {
-    delete rootProject.buildDir
-}
+    id 'com.android.application' version '8.0.0-beta04' apply false
```
[the remark this answers is not in the document]
Comment (android-dev-lxl) [reply build.gradle:3]: sure

**Review by kkuan2011 — APPROVED** [review]

File: app/build.gradle (hunk 1 above)
Comment (yenerm) [reply app/build.gradle:1]: Looks like we accidentaly merged this without a license

File: settings.gradle [hunk 7]
```diff
@@ -1,24 +1,8 @@
-/*
```
Comment (yenerm) [inline settings.gradle:1]: license is accidentally removed

File: gradle/wrapper/gradle-wrapper.properties [hunk 8]
```diff
@@ -1,21 +1,22 @@
 #
```
Comment (yenerm) [inline gradle/wrapper/gradle-wrapper.properties:1]: nit: no license needed on .properties files

**Pull request merged**